subprojects/logging/src/main/java/org/gradle/internal/logging/buildoperation/StyledTextBuildOperationProgressDetails.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.logging.buildoperation;
+
+import org.gradle.internal.logging.text.StyledTextOutput;
+import org.gradle.internal.scan.UsedByScanPlugin;
+
+import java.util.List;
+
+public interface StyledTextBuildOperationProgressDetails extends OutputBuildOperationProgressDetails{
+    List<? extends SpanDetail> getSpanDetails();
```

Comment:
Please rename this to getSpans().

Reply:
I named it like this as getSpans() conflicts with `StyledTextOutputEvent#getSpans()`

Reply:
changed